mount.go

```diff
@@ -374,3 +379,36 @@ func addStorages(ctx context.Context, storages []*pb.Storage, s *sandbox) (mount
 
 	return mountList, nil
 }
+
+// getMountFSType returns the FS type corresponding to the passed mount point and
+// any error ecountered.
+func getMountFSType(mountPoint string) (string, error) {
```

Comment:
Hi @marcov - please could you write a unit test for this function. This function is perfectly suited to a [table-based](https://github.com/kata-containers/tests/blob/master/Unit-Test-Advice.md#table-driven-tests) approach fwics.

Reply:
Thanks for almost writing a test for me :D